app/src/controllers/userController.ts

```diff
@@ -0,0 +1,60 @@
+import mongoose from 'mongoose';
+import { UserSchema } from '../models/userModel';
+import { Request, Response } from 'express';
+
+const User = mongoose.model('User', UserSchema);
+
+export class UserController {
+
+    public async index (req: Request, res: Response) {
+        try {
+            const users = await User.find();
+            res.json(users);
+        } catch(err) {
+            res.send(err);
```

Comment:
API can't send HTTP 4xx or 5xx. You need to send JSON with reason why error occured.

For example, 

Success
```js
{
    data: { // actions payload
    },
    statusCode: 200
}
```

Errors:
```js
{
    statusCode: 4xx|5xx,
    errorMessage: ""
}
```